components/brave_wallet_ui/line_chart/line_chart.html

```diff
@@ -0,0 +1,25 @@
+<!--Copyright (c) 2023 The Brave Authors. All rights reserved.-->
+<!--This Source Code Form is subject to the terms of the Mozilla Public-->
+<!--License, v. 2.0. If a copy of the MPL was not distributed with this file,-->
+<!--you can obtain one at http://mozilla.org/MPL/2.0/.-->
+
+<!doctype html>
+<head>
+   <meta charset="UTF-8">
+   <title>Line Chart iframe</title>
+   <script src="chrome-untrusted://resources/js/load_time_data_deprecated.js"></script>
```

Comment:
I believe there's a new non deprecated resource we should be using here with .m.js

Reply:
Removed. It was not needed